routes/authRoutes.js

```diff
@@ -3,22 +3,30 @@ const { UserModel } = require('../models/Users'); // Import User model
 const { ContentCreatorModel } = require('../models/ContentCreators'); // Import Content Creator model
 const { InstitutionModel } = require('../models/Institutions'); //Import Institution model 
 const { StudentModel } = require('../models/Students'); // Import Student model
-
+const { ApplicationModel } = require('../models/Applications'); // Import Applications model
 const { makeExpressCallback } = require('../helpers/express');
 const { authEndpointHandler } = require('../auth');
 const { signAccessToken } = require('../helpers/token');
 const { compare, encrypt } = require('../helpers/password');
 const errorCodes = require('../helpers/errorCodes');
 const { sendResetPasswordEmail } = require('../helpers/email');
 const { PasswordResetToken } = require('../models/PasswordResetToken');
+const { EmailVerificationToken } = require('../models/EmailVerificationToken');
 const { validateEmail, validateName, validatePassword } = require('../helpers/validation');
+const { sendVerificationEmail } = require('../helpers/email');
 
+const bcrypt = require('bcrypt');
+/* const { userList } = require('../users'); */
+// Utility function to encrypt the token or password
 const TOKEN_EXPIRATION_TIME = 1000 * 60 * 5;
-const ATTEMPT_EXPIRATION_TIME = 1000 * 60 * 5; //1000 * 60 * 60;
+const ATTEMPT_EXPIRATION_TIME = 1000 * 60 * 5;//1000 * 60 * 60;
 
 // Services
 //require("../services/passport");
```

Comment:
unused code should be removed also applies to line 19.
Small things that can be ignored for now but good to get in the habit of for later in the project